### PR TITLE
feat/add-replace-param-props-codemod

### PR DESCRIPTION
**Pull request by DmytroHryshyn**
https://linear.app/codemod/issue/CDMD-2461/[codemod-new]-replace-param-prop

### CODE REVIEW

File: apps/registry/codemods/react-router/4/replace-param-prop/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const PostEdit = ({ match }) => (
+	<div>
+		<h1>Post {match.params.id}</h1>
+	</div>
+);
+```
+
+## Applicability Criteria
+
+react-router version 3
```
Comment (hbjORbj) [inline apps/registry/codemods/react-router/4/replace-param-prop/README.md:31]: React Router version 3.x.y

File: apps/registry/codemods/next/13/new-link/README.md [hunk 2]
```diff
@@ -9,21 +9,21 @@ Safely removes `<a>` from `Link` components imported from the `next/link` module
...
+);
 ```
 
 ### After
 
 ```jsx
-export default function Component() {
-	return <Link href="/a">Anchor</Link>;
-}
+const PostEdit = ({ match }) => (
```
Comment (hbjORbj) [inline apps/registry/codemods/next/13/new-link/README.md:22]: wrong file?

**Comment by hbjORbj** [comment]
let's add some more test cases

File: apps/registry/codemods/react-router/4/replace-param-prop/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Other Metadata
+
+### Codemod Version
+
+v1.0.0
+
+### Change Mode
+
+**Assistive**: The automation partially completes changes. Human involvement is needed to make changes ready to be pushed and merged.
```
[the remark this answers is not in the document]
Comment (DmytroHryshyn) [reply apps/registry/codemods/react-router/4/replace-param-prop/README.md:41]: to make it autonomous we need to cover more use cases. currently i've added support for the basic use case

**Review by hbjORbj — APPROVED** [review]

**Pull request merged**